vector/src/main/java/im/vector/app/features/home/room/list/RoomListViewModel.kt

```diff
@@ -181,7 +173,7 @@ class RoomListViewModel @AssistedInject constructor(
         return session.getRoom(roomId)?.stateService()?.isPublic().orFalse()
     }
 
-    fun deleteLocalRooms(roomsIds: Set<String>) {
+    fun deleteLocalRooms(roomsIds: List<String>) {
```

Comment:
Using `Iterable` could help to not depend on the implementation

```suggestion
    fun deleteLocalRooms(roomsIds: Iterable<String>) {
```

Reply:
Done https://github.com/vector-im/element-android/pull/7123/commits/6c9ae331e324edc20dd6d9bcf7c64db20b41c46c